### ISSUE TITLE
RHEL7 does not need the collectd-python package

### ISSUE THREAD

**Issue by tarjei** [pdxcat/puppet-module-collectd#239]
It is included by default.

Thanks for a great module btw!

**Comment by tarjei** [comment]
```diff
--- a/modules/collectd/manifests/plugin/python.pp
+++ b/modules/collectd/manifests/plugin/python.pp
@@ -12,9 +12,9 @@ define collectd::plugin::python (
   validate_hash($config)
 
   if $::osfamily == 'Redhat' {
-    package { 'collectd-python':
-      ensure => $ensure,
-    }
+#    package { 'collectd-python':
+#      ensure => $ensure,
+#    }
   }
 
```

**Comment by nibalizer** [comment]
Can you format this as a pull request?

**Comment by jesusaurus** [comment]
Also, can you remove the package resource rather than commenting it out?

**Comment by blkperl** [comment]
You need to add a conditional on the version of redhat around the package resource.

**Comment by tarjei** [comment]
This wasn't actually meant as a PR, put as a tip to where to change the code :)

**Comment by bzed** [comment]
Using ensure_packages from stdlib would be very appreciated....

**Comment by blkperl** [comment]
Released in #254
